linux_os/guide/system/accounts/accounts-session/accounts_users_home_files_permissions/bash/shared.sh

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+awk -F':' '{ if ($4 >= {{{ uid_min }}} && $4 != 65534) system("chmod -Rf 700 "$6) }' /etc/passwd
```

Comment:
Besides it setting the permission to `700` instead of `750`.
The remediation is changing the permissions of all files, but I think it should only change the permission of files that were not correct. If a file had a more strict permission than `700` (`750`) it would be loosened.

I think the same applies to Ansible.

Reply:
Update is cumming. : ))

Reply:
Update is coming. : ))